book/src/install.md

```diff
@@ -134,7 +134,9 @@ pacman -S mingw-w64-ucrt-x86_64-helix
 
 ## Building from source
 
-Clone the repository:
+Clone the Helix GitHub repository into a directory of your choice. The
+examples in this documentation assume installation into either `~/src/` on
+Linux and macOS, or `%userprofile%\src\helix\runtime` on Windows.
```

Comment:
I like it! In https://github.com/helix-editor/helix/pull/6809 there is now a list format:

1. Clone the repository:

You could put your new paragraph under that, or turn `Clone the repository:` and `Compile from source:` into sub headings... I think I had them subheadings originally, but the maintainers didn't like it, @the-mikedavis will know best :)

Reply:
@David-Else Should we merge these to PRs or should I wait until #6809 is merged into master?